### PR TITLE
v0.5.0

### PR DESCRIPTION
**Pull request by benpatterson**
TE-1218

### CODE REVIEW

**Comment by benpatterson** [comment]
@jzoldak @dianakhuang @cahrens 

Set to 0.5.0 and rebased on master.

**Comment by cahrens** [comment]
:+1: FYI @efischer19

**Comment by efischer19** [comment]
Awesome, thanks all.

I can update the version installed in edx-platform via https://github.com/edx/edx-platform/pull/11758

**Comment by benpatterson** [comment]
@efischer19 how close is that PR to being released? I also need bok-choy updates for some of the work I'm doing on platform. Wanted a sense of timing...

**Comment by cahrens** [comment]
@benpatterson The plan is to merge it tomorrow (it must be in by Wednesday for the "marathon").

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,17 @@
+v0.5.0 (3/22/16)
+* Added `scroll_to_element(element_selector)`
+* Make bok-choy more resilient to transient webdriver failures
```
Comment (jzoldak) [inline CHANGELOG]: nit: Make browser startup more resilient...?

File: CHANGELOG [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+* Disable default promo tabs on firefox profile
+* Optional XSS vulnerability check
```
Comment (jzoldak) [inline CHANGELOG]: nit: that might minimize the new feature's usefulness. Maybe "Implement feature for XSS (cross-site scripting) vulnerability auditing"

File: CHANGELOG [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+* Updates for contributors (i.e. we use Slack now, not IRC)
+* Various improvements for how the bok-choy repo itself is tested
+* Python version compatibility improvements
+
+v0.4.11 (n/a)
+* Skipped this release version due to infrastructure udpates (not releasing via Travis)
```
Comment (jzoldak) [inline CHANGELOG]: typo s/udpates/updates/

**Comment by jzoldak** [comment]
:+1:

**Comment by benpatterson** [comment]
Updated/squashed.

**Pull request merged**